hack/vendor/olm-test-script/e2e-olm.sh

```diff
@@ -83,26 +95,23 @@ $CSV
   packages: |-
 $PKG
 EOF
-CSV_CHANNEL=$(sed -nr 's,.*name: \"?([^"][^"]*)\"?,\1,p' $MANIFEST_DIR/*package.yaml)
-CURRENT_CSV=$(sed -nr 's,.*currentCSV: (.*),\1,p' $MANIFEST_DIR/*package.yaml)
-PACKAGE_NAME=$(sed -nr 's,.*packageName: (.*),\1,p' $MANIFEST_DIR/*package.yaml)
+
+CSV_CHANNEL=$($GNUSED -nr 's,.*name: \"?([^"][^"]*)\"?,\1,p' $MANIFEST_DIR/*package.yaml)
+CURRENT_CSV=$($GNUSED -nr 's,.*currentCSV: (.*),\1,p' $MANIFEST_DIR/*package.yaml)
+PACKAGE_NAME=$($GNUSED -nr 's,.*packageName: (.*),\1,p' $MANIFEST_DIR/*package.yaml)
 
 oc create -n $TEST_NAMESPACE -f /tmp/configmap.yaml
 if [ "${CREATE_OPERATORGROUP}" == "true" ] ; then
-  if [ "${GLOBAL}" == "true" ] ; then
-    oc -n ${TARGET_NAMESPACE} process -f "$(dirname $0)/operatorgroup-template-global.yaml" -p SUFFIX="${SUFFIX:-}" | oc create -n $TEST_NAMESPACE -f -
+  if [ "${TARGET_NAMESPACE}" = all ] ; then
```

Comment:
We need to be careful about renaming of these vars and the logic here as there may be CI and other places that will require updating. Also this particular script we need to not drift too far from its original source since we should contribute it back to https://github.com/shawn-hurley/olm-test-script  This will eventually be replaced by actual tooling

Reply:
The actual rev of this file is from https://github.com/shawn-hurley/olm-test-script/pull/5

Reply:
Removed the script updates, as the shawn-hurley/olm-test-script#5 is not urgent and not merged yet.